src/journal-remote/microhttpd-util.c

```diff
@@ -72,20 +72,25 @@ int mhd_respond(struct MHD_Connection *connection,
                 enum MHD_RequestTerminationCode code,
                 const char *message) {
 
+        const char *fmt;
+
+        fmt = strjoina(message, "\n");
+
         return mhd_respond_internal(connection, code,
-                                    (char*) message, strlen(message),
+                                    fmt, strlen(message) + 1,
```

Comment:
nitpick: I think `strlen(fmt)` would be less error prone, in case `fmt` ever changes.